forge-gui/res/cardsfolder/upcoming/filigree_vector.txt

```diff
@@ -0,0 +1,17 @@
+Name:Filigree Vector
+ManaCost:3 W
+Types:Artifact Creature Phyrexian Construct
+PT:1/1
+T:Mode$ ChangesZone | Origin$ Any | Destination$ Battlefield | ValidCard$ Card.Self | Execute$ TrigCounter | TriggerDescription$ When CARDNAME enters the battlefield, put a +1/+1 counter on each of any number of target creatures and a charge counter on each of any number of target artifacts.
+SVar:TrigCounter:DB$ Pump | ValidTgts$ Creature | AILogic$ Pump | TgtPrompt$ Select any number of target creatures to distribute +1/+1 counters to | TargetMin$ 0 | TargetMax$ X | RememberTargets$ True | SubAbility$ DBChargeCounter
+SVar:DBChargeCounter:DB$ Pump | ValidTgts$ Artifact | AILogic$ Pump | TgtPrompt$ Select any number of target artifacts to distribute charge counters to | TargetMin$ 0 | TargetMax$ Y | ImprintCards$ ThisTargetedCard | SubAbility$ DBPutCounterAll
```

Comment:
Without digging too deep into the PumpAi my guess is it isn't wired up handle "Pump" on artifacts...

Maybe try one more thing before we put the RemoveDeck labels on there... add `AITgts$ Artifact.YouCtrl`

Reply:
Oh seems to work now. I'll keep AITgts$ in mind for the future
![image](https://user-images.githubusercontent.com/67333662/235352390-c89ad15d-bc94-4f4d-9e64-59a94553d70a.png)